Cargo.toml

```diff
@@ -29,9 +32,20 @@ path = "crossbeam-ebr/crossbeam-epoch"
 package = "crossbeam-pebr-epoch"
 path = "crossbeam-pebr/crossbeam-epoch"
 
-# TODO: remove this
 [profile.release]
+lto = true
+codegen-units = 1
+panic = "abort"
+
+[profile.release-with-debug]
+inherits = "release"
+debug = true
+
+[profile.release-simple]
+inherits = "release"
 debug = true
+lto = false
+codegen-units = 16
```

Comment:
why?

Reply:
That `release-simple` profile is intended to be used for simple release mode debugging and sanitizing. We are using the link-time optimization for the production build, but its compilation is too slow to frequently debug the (optimized) code with Address sanitizer. For this reason, I have added that option, turning off the LTO and increasing codegen-units to make the release build for sanitizing faster.